boards/arm/mps2_an521/CMakeLists.txt

```diff
@@ -7,3 +7,24 @@
 zephyr_library()
 zephyr_library_sources(pinmux.c)
 zephyr_library_include_directories(${ZEPHYR_BASE}/drivers)
+
+if(CONFIG_SOC_MPS2_AN521_CPU1)
+# Building a firmware image for CPU1: this requires a binary
+# for CPU0, which will boot the device and wake up CPU1.
+
+  set(CPU0_BINARY_DIR ${BOARD_DIR}/empty_cpu0-prefix/src/empty-cpu0-build/zephyr)
+
+  enable_language(C ASM)
```

Comment:
why is this needed here ?

Reply:
possibly some redundant copy-paste
will check.

Reply:
Removed now, thanks!